src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -284,6 +285,95 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
     }
     
+    private void testEditVisibilityAction() throws Exception {
+
+        ______TS("edit question 1 to Giver's team members and Giver");
```

Comment:
The row of Giver's Team Members and Giver is with tag <hide style="display:none"> which means this row does not be displayed at all (different from hidden), not even be located the space for it. I am not sure how to use Selenium to find this content of this row. I use the css selector but it show exception that this element (the row) is not visible and not interactive. 
This is how the html looks like: 
<img width="892" alt="screen shot 2016-02-29 at 12 34 29" src="https://cloud.githubusercontent.com/assets/13346928/13386005/5efbd568-dee2-11e5-9a29-8c290152e9df.png">
Is there any better way to verify this?